include/cista/hashing.h

```diff
@@ -6,7 +6,6 @@
 #include <string>
 #include <string_view>
 #include <tuple>
-#include <type_traits>
```

Comment:
Please leave the `<type_traits>` include there. The fact that `cista/type_traits.h` includes `<type_traits>` is an implementation detail (which means this might change in the future). It's more robust to "include what you use" and there are lots of other places in `hashing.h` that use stuff from `<type_traits>`.